tests/dict/DictSelectTest.php

```diff
@@ -40,7 +40,7 @@ public static function provideTestDiffByKey(): varray<mixed> {
         dict[2 => 4, 4 => 8, 8 => 16],
         varray[
           Map {1 => 1, 2 => 2},
-          HackLibTestTraversables::getKeyedIterator(Vec\range(0, 3)),
```

Comment:
Why did this change happen? This isn't testing the same thing any more...

If this was generated automatically, there might be a bug in the script.

Reply:
Oh I see now, a bunch of HSL functions were changed to take Containers instead of Traversables, but the tests were not fixed accordingly. So this was a legitimate issue that we detected thanks to your new lint rule!

Reply:
There are a couple of places where I changed the following things.

_`getIterator()` -> `getKeyedIterator()`_
The signature of the hsl function under test requires a KeyedTraversable, not a Traversable.
At runtime, these two are the same object, so nothing has really changed.

_`getKeyedIterator(<%keyedconatiner%>)`_ -> `keyedcontainer`_
Some hsl functions used to take `Traversable` / `KeyedTraversable` arguments, but this was changes to `Container` and `KeyedContainer` some time ago, because of reactivity changes.

![image](https://user-images.githubusercontent.com/31805625/79811982-cce30b80-8377-11ea-8b65-c81ba6473427.png)
d3019f237df9de229d9fc86e2da4b1546517b2e9

Reply:
Somehow I did not see your `Oh I see ... lint rule!` response until sending my own.
We independently came to the same conclusion.